go/cmd/dolt/commands/commit.go

```diff
@@ -123,7 +123,7 @@ func (cmd CommitCmd) Exec(ctx context.Context, commandStr string, args []string,
 		}
 	}
 
-	err = actions.CommitStaged(ctx, dEnv, actions.CommitStagedProps{
+	err = actions.CommitStaged(ctx, dEnv.DoltDB, dEnv.RepoStateReader(), dEnv.RepoStateWriter(), actions.CommitStagedProps{
```

Comment:
I think it might not be worth it since there will a large refactor whenever we think there is going to be multiple writers. I also like having all params listed as that clearly states that this function is either modifying or just reading.

Reply:
yeah that's reasonable